gensrc/thrift/PlanNodes.thrift

```diff
@@ -406,6 +406,7 @@ struct TMergeJoinNode {
 }
 
 enum TAggregationOp {
+  ANY_VALUE,
```

Comment:
The new addition needs to be put at the end so that forward compatibility can be achieved.